### PR TITLE
doc: release-notes-3.7: add release notes for USB

### PR DESCRIPTION
**Pull request by jfischer-no**
Add release notes for USB support.

### CODE REVIEW

File: doc/releases/release-notes-3.7.rst [hunk 1]
```diff
@@ -1292,6 +1296,13 @@ Networking
 USB
 ***
 
+* new USB device stack:
+
+  * Added support for HID devices
```
Comment (tmon-nordic) [inline doc/releases/release-notes-3.7.rst:1301]: UAC2 is also brand new class implemented in Zephyr 3.7.0

**Review by tmon-nordic — CHANGES_REQUESTED** [review]
Is `struct usbd_context` -> `struct usbd_contex` rename worhwhile to mention in the release notes?

File: doc/releases/release-notes-3.7.rst (hunk 1 above)
Comment (jfischer-no) [reply doc/releases/release-notes-3.7.rst:1301]: https://docs.zephyrproject.org/latest/releases/release-notes-3.6.html#usb

**Comment by jfischer-no** [comment]
> Is `struct usbd_context` -> `struct usbd_contex` rename worhwhile to mention in the release notes?

I am not sure, I only mention major changes, for everything else there is the git log.

File: doc/releases/release-notes-3.7.rst (hunk 1 above)
Comment (tmon-nordic) [reply doc/releases/release-notes-3.7.rst:1301]: I just double checked and I somehow messed it up yesterday. c83ad53442723a7412e8ae060f5d8bdf142f0274 was indeed before 3.6.0 and 5e3a73de04b99cc25deeb64570bbfc95b0a654df was after 3.6.0.

**Review by tmon-nordic — APPROVED** [review]

**Review by aescolar — APPROVED** [review]

**Pull request merged**